src/Tribe/Tickets_View.php

```diff
@@ -0,0 +1,516 @@
+	public function inject_link_template_the_content( $content ) {
+		$event_id = get_the_ID();
+		$user_id = get_current_user_id();
+
+		// If we have this we are already on the tickets page
+		$is_correct_page = get_query_var( 'tribe-edit-orders', false );
+		if ( $is_correct_page ) {
+			return $content;
+		}
+
+		if ( ! $this->has_rsvp_attendees( $event_id, $user_id ) && ! $this->has_rsvp_attendees( $event_id, $user_id ) ) {
+			return $content;
+		}
+
+		ob_start();
+		include Tribe__Tickets__Templates::get_template_hierarchy( 'tickets/orders-link.php' );
+		$content .= ob_get_clean();
+
+		return $content;
+	}
+
+	/**
+	 * Fetches from the Cached attendees list the ones that are relevant for this user and event
+	 * Important to note that this method will bring the attendees organized by order id
+	 *
+	 * @param  int       $event_id      The Event ID it relates to
+	 * @param  int|null  $user_id       An Optional User ID
+	 * @param  boolean   $include_rsvp  If this should include RSVP, which by default is false
+	 * @return array                    List of Attendees grouped by order id
+	 */
+	public function get_event_attendees_by_order( $event_id, $user_id = null, $include_rsvp = false ) {
+		$attendees = Tribe__Tickets__Tickets::get_event_attendees( $event_id );
+		$orders = array();
+
+		foreach ( $attendees as $key => $attendee ) {
+			// Ignore RSVP if we don't tell it specifically
+			if ( 'rsvp' === $attendee['provider_slug'] && ! $include_rsvp ) {
+				continue;
+			}
+
+			// If we have a user_id then test it and ignore the ones that don't have it
+			if ( ! is_null( $user_id ) ) {
+				if ( empty( $attendee['user_id'] ) || $attendee['user_id'] != $user_id ) {
+					continue;
+				}
+			}
+
+			$orders[ (int) $attendee['order_id'] ][] = $attendee;
+		}
+
+		return $orders;
+	}
+
+	/**
+	 * Fetches from the Cached attendees list the ones that are relevant for this user and event
+	 * Important to note that this method will bring the attendees from RSVP
+	 *
+	 * @param  int       $event_id     The Event ID it relates to
+	 * @param  int|null  $user_id      An Optional User ID
+	 * @return array                   Array with the RSVP attendees
+	 */
+	public function get_event_rsvp_attendees( $event_id, $user_id = null ) {
+		$all_attendees = Tribe__Tickets__Tickets::get_event_attendees( $event_id );
+		$attendees = array();
+
+		foreach ( $all_attendees as $key => $attendee ) {
+			// Skip Non RSVP
+			if ( 'rsvp' !== $attendee['provider_slug'] ) {
+				continue;
+			}
+
+			// If we have a user_id then test it and ignore the ones that don't have it
+			if ( ! is_null( $user_id ) ) {
+				if ( empty( $attendee['user_id'] ) || $attendee['user_id'] != $user_id ) {
+					continue;
+				}
+			}
+
+			$attendees[] = $attendee;
+		}
+
+		return $attendees;
+	}
+
+	/**
+	 * Gets a List of Possible RSVP answers
+	 * @return array
+	 */
+	public function get_rsvp_options() {
+		$options = array(
+			'yes' => __( 'Going', 'event-tickets' ),
```

Comment:
So I think we present 'yes' as 'Going' on the frontend (and 'no' as 'Not Going') but in the attendee list we expose the raw key name, ie 'yes' or 'no', under order status... should we keep things consistent there and use 'Going' or 'Not Going' also? Those terms make more sense to me than an order status of yes.